lnurl-withdraw.md

```diff
@@ -13,12 +13,13 @@ Today users are asked to provide a withdrawal Lightning invoice to a service, th
 3. `LN WALLET` gets JSON response from `LN SERVICE` of form:
     ```
     {
+        tag: "withdrawRequest", // type of LNURL
         callback: String, // the URL which LN SERVICE would accept a withdrawal Lightning invoice as query parameter
         k1: String, // random or non-random string to identify the user's LN WALLET when using the callback URL
-        maxWithdrawable: MilliSatoshi, // max withdrawable amount for a given user on LN SERVICE
         defaultDescription: String, // A default withdrawal invoice description
-        minWithdrawable: MilliSatoshi // An optional field, defaults to 1 MilliSatoshi if not present, can not be less than 1 or more than `maxWithdrawable`
-        tag: "withdrawRequest" // type of LNURL
+        maxWithdrawable: MilliSatoshi, // max withdrawable amount for a given user on LN SERVICE
+        minWithdrawable: MilliSatoshi, // An optional field, defaults to 1 MilliSatoshi if not present, can not be less than 1 or more than `maxWithdrawable`
```

Comment:
Should be allowed to be zero. The use case is in ATMs which want to make sure users wallet supports this before allowing to insert money. This avoids having to process returns of money.

Reply:
I'm not sure about that very specific use case, but I think it makes sense to allow zero, specially considering automatic `balanceCheck`s.